### ISSUE TITLE
Should babili also enable all the finalized spec parse features?

### ISSUE THREAD

**Issue by thejameskyle** [babel/babili#133]
Seems like it should

**Comment by hzoo** [comment]
Is it erroring currently? In the new babylon, we moved the es2017 plugins on by default https://github.com/babel/babylon/releases/tag/v6.9.1

**Comment by thejameskyle** [comment]
Oh maybe this is just an issue with the REPL on the babel site.

**Comment by hzoo** [comment]
since we are using babel-standalone it might be that either the repl/that doesn't have the latest parser in the bundle

**Comment by Daniel15** [comment]
babel-standalone just bundles the latest Babel + dependencies, is there something special I need to enable? You can enable the es2017 preset in the REPL, it's just not enabled by default.

**Comment by hzoo** [comment]
If we update the parser (babylon), we need to update babel-standalone again right? I'm not sure but maybe it was updated (7 days ago) after so it doesn't have the changes

**Comment by Daniel15** [comment]
Good catch... The version number of babel-standalone is purely based off the Babel version. It uses the maximum version number of all the npm dependencies prefixed with `babel-`. If Bablyon is updated but the version of Babel doesn't change, a new version of babel-standalone won't be pushed either. Perhaps I should change the versioning scheme of Babel-standalone to handle Babylon updates too.

Having said that, Babel-standalone 6.14.0 was posted a few days ago, so it should include the latest Babylon (unless npm messed up, which is common)
